app/helpers/textual_mixins/textual_metrics_status.rb

```diff
@@ -0,0 +1,39 @@
+module TextualMixins::TextualMetricsStatus
+  def textual_time_ago(time)
+    if time
+      time_str = time_ago_in_words(time.in_time_zone(Time.zone)).titleize
+      _("%{time} Ago") % { :time => time_str }
+    else
+      _("Never")
+    end
+  end
+
+  def textual_authentication_value(status, updated_on, valid_str = _("Valid"))
```

Comment:
Are we sure that this is the right place for this function? It seems good and helpful but I am not sure that we want to keep it in this file / module. Same goes for ```textual_authentication_value```. Maybe we should have a ```textual_authentication_status``` file as well?